.pre-commit-config.yaml

```diff
@@ -1,73 +1,32 @@
-fail_fast: true
-
+# Keep tool versions in sync with the versions in requirements-dev.txt
 default_language_version:
-  python: python3
-
-exclude: ^.*\b(migrations)\b.*$
-default_stages: [commit, manual]
-
+    python: python3
+default_install_hook_types: [pre-commit, commit-msg]
+default_stages: [pre-commit, manual]
 repos:
-  - repo: https://github.com/psf/black
-    rev: 23.1.0
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v6.0.0
     hooks:
-      - id: black
-        name: (BF) black
-        files: ^backend/benefit
-      - id: black
-        name: (KS) black
-        files: ^backend/kesaseteli
-      - id: black
-        name: (Shared) black
-        files: ^backend/shared
-  - repo: https://github.com/pycqa/isort
-    rev: 5.12.0
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
+      - id: check-toml
+      - id: check-added-large-files
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.13.1
     hooks:
-      - id: isort
-        name: (BF) isort
-        args: [--settings-file, backend/benefit/setup.cfg]
-        files: ^backend/benefit
-      - id: isort
-        name: (KS) isort
-        args: [--settings-file, backend/kesaseteli/setup.cfg]
-        files: ^backend/kesaseteli
-      - id: isort
-        name: (Shared) isort
-        args: [--settings-file, backend/shared/setup.cfg]
-        files: ^backend/shared
-  - repo: https://github.com/pycqa/flake8
-    rev: 6.0.0
+      # Run the linter.
+      - id: ruff
```

Comment:
`ruff` id has become legacy and is replaced with `ruff-check`

Reply:
```suggestion
      - id: ruff-check
```

Reply:
fixed in 950b2252702910b06c82005a53390aceba3af511